share/completions/just.fish

```diff
@@ -0,0 +1,4 @@
+function _justfile_targets
+    just --summary 2>/dev/null | string split ' ' | string match -v '.*/.*'
```

Comment:
That last `string match` is in glob-mode, so you'll filter out any lines that have a period, then something, a "/." and then something.

Is that what you want or do you need to add `-r`?

Reply:
I removed the match altogether since it wasn't doing anything. just a rememnant